easybuild/main.py

```diff
@@ -402,7 +402,11 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
 
     # creating/updating PRs
     if new_update_pr:
-        if options.new_pr:
+        if options.preview_pr:
+            print preview_pr(paths, colored=use_color(options.color))
+            cleanup(logfile, eb_tmpdir, testing, silent=True)
+            sys.exit(0)
```

Comment:
Hmm, can we avoid doing cleanup/exit here? How is this handled for `--new-pr`?

Reply:
I think it's because `options.new_pr` is in `no_ec_opts`, there's a cleanup later for those, I suppose `options.preview_pr` can also be added to them (although it's a bit counterintuitive that these don't require any easyconfigs to be specified?)